os/include/tinyara/wdog.h

```diff
@@ -180,6 +180,11 @@ int wd_start(WDOG_ID wdog, int delay, wdentry_t wdentry, int argc, ...);
 int wd_cancel(WDOG_ID wdog);
 int wd_gettime(WDOG_ID wdog);
 
+#ifdef CONFIG_SCHED_TICKSUPPRESS
+int wd_getdelay(void);
+void wd_updatedelay(int elapsed);
```

Comment:
What if a timer offers 64 bit registers, then the max threshold may change to 2 power 64.  should we consider that or assume that it's always 32 bit register and use int as its data type

Reply:
As already defined in **include/tinyara/wdog.h**, wdog delay itself is limited by type '**int**' . https://github.com/Samsung/TizenRT/blob/8cb7c62c626e7f37c7ab72413444024785ca9942/os/include/tinyara/wdog.h#L152